### PR TITLE
 Introduce VM version ('hard fork') column for assembly opcodes

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: docs/julia.rst [hunk 1]
```diff
@@ -320,168 +320,169 @@ The following functions must be available:
...
 | delegatecall(g:u256, a:u256, in:u256,       | identical to ``callcode``,                                      |
 | insize:u256, out:u256,                      | but also keep ``caller``                                        |
 | outsize:u256) -> r:u256                     | and ``callvalue``                                               |
-+---------------------------------------------------------------------------------------------------------------+
++---------------------------------------------+-----------------------------------------------------------------+
 | stop()                                      | stop execution, identical to return(0,0)                        |
-| Perhaps it would make sense retiring this as it equals to return(0,0). It can be an optimisation by the EVM   |
-| backend.                                                                                                      |
-+---------------------------------------------------------------------------------------------------------------+
+|                                             | Perhaps it would make sense retiring this as it equals to       |
```
Comment (chriseth) [inline docs/julia.rst:409]: Where did this comment come from?

File: docs/assembly.rst [hunk 2]
```diff
@@ -153,6 +153,7 @@ If an opcode takes arguments (always from the top of the stack), they are given
 Note that the order of arguments can be seen to be reversed in non-functional style (explained below).
 Opcodes marked with ``-`` do not push an item onto the stack, those marked with ``*`` are
 special and all others push exactly one item onto the stack.
+Opcodes marked with ``H``, ``B`` or ``C`` are present since Homestead, Byzantium or Constantinople, respectively.
```
Comment (chriseth) [inline docs/assembly.rst]: Perhaps the column has better visibility if we do not mark opcodes at all which are available even before homestead. We should also mention that constantinople is still in planning.

File: docs/julia.rst (hunk 1 above)
Comment (axic) [reply docs/julia.rst:409]: It was there. (This is the julia part.)

File: docs/assembly.rst (hunk 2 above)
Comment (axic) [reply docs/assembly.rst]: Oh right, its only `delegatecall` for homestead, everything else is frontier.

File: docs/julia.rst (hunk 1 above)
Comment (chriseth) [reply docs/julia.rst:409]: I know, but I still think this has no place in a documentation :)

Comment (axic) [reply docs/julia.rst:409]: There's another PR which updates the builtins #3238, I'd just clean up these there and leave this single change as a formatting fix.

File: docs/assembly.rst (hunk 2 above)
Comment (axic) [reply docs/assembly.rst]: Updated (also fixed `revert`). Do we want to keep `F` for frontier or remove it?

**Review by chriseth — APPROVED** [review]

**Pull request merged**